static/app/views/alerts/rules/issue/ruleNode.tsx

```diff
@@ -374,8 +374,8 @@ function RuleNode({
               <TicketRuleModal
                 {...deps}
                 formFields={node.formFields || {}}
-                link={node.link}
-                ticketType={node.ticketType}
+                link={node.link!}
+                ticketType={node.ticketType!}
```

Comment:
`link` and `ticketType` will always be set for this action type which you can see below.
https://github.com/getsentry/sentry/blob/284188874cef378a167cdb97f9fe0b4d0c407431/src/sentry/api/endpoints/project_rules_configuration.py#L57-L60